doc/source/getting_started/contribution.rst

```diff
@@ -4,106 +4,71 @@
 Contributing
 ============
 
+
+.. toctree::
+   :hidden:
+   :maxdepth: 3
+
+   write_documentation
+   develop_pymapdl
+
+
+There are several ways to contribute to PyMAPDL library. 
+
+* `Answering discussions`_
+* `Posting issues`_
+* :ref:`write_documentation`
+* :ref:`developing_pymapdl`
+
 Overall guidance on contributing to a PyAnsys library appears in the
 `Contributing <dev_guide_contributing_>`_ topic
 in the *PyAnsys Developer's Guide*. Ensure that you are thoroughly familiar
 with it and all `Coding style <dev_guide_coding_style_>`_ before attempting to
 contribute to PyMAPDL.
  
-The following contribution information is specific to PyMAPDL.
-
-
-Cloning the PyMAPDL repository
-==============================
 
-Run this code to clone and install the latest version of PyMAPDL in development mode:
+Answering discussions
+=====================
 
-.. code:: console
+Answering discussions in a excellent way to contribute to PyMAPDL project, and
+it does not require any setup, just a GitHub account.
+It is probably the first step towards becoming a full PyMAPDL developer, since it
+helps you deepen your understanding of the project, as engaging in
+discussions often requires a thorough grasp of the project's goals and
+challenges.
+Your contributions can help other users or contributors who
+may be facing similar issues, making the repository more welcoming and
+inclusive. By providing answers or solution, you can directly contribute to the project's
+success, maintain its health, and encourage a positive, open source ecosystem.
```

Comment:
```suggestion
It is probably the first step towards becoming a full PyMAPDL developer, since it
helps you deepen your understanding of the project. Engaging in
discussions often requires a thorough grasp of the project's goals and
challenges.
Your contributions can help other users or contributors who
may be facing similar issues, making the repository more welcoming and
inclusive. By providing answers or solutions, you can directly contribute to the project's
success, maintain its health, and encourage a positive, open source ecosystem.
```
Bravo!